packages/paste-website/src/pages/components/skeleton-loader/index.mdx

```diff
@@ -0,0 +1,233 @@
+---
+title: Skeleton Loader - Components
+description: A component that gives users a hint of what type of information will be loaded on a page and helps reduce the perceived time a page takes to load.
+slug: /components/skeleton-loader/
+---
+
+import {graphql} from 'gatsby';
+import {Anchor} from '@twilio-paste/anchor';
+import {Avatar} from '@twilio-paste/avatar';
+import {Box} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+import {Card} from '@twilio-paste/card';
+import {Heading} from '@twilio-paste/heading';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {SkeletonLoader} from '@twilio-paste/skeleton-loader';
+import {Stack} from '@twilio-paste/stack';
+import {Table, THead, TBody, Tr, Td, Th} from '@twilio-paste/table';
+import {Text} from '@twilio-paste/text';
+import {CalendarIcon} from '@twilio-paste/icons/esm/CalendarIcon';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {
+  CompositionExample,
+  TextExample,
+  ParagraphExample,
+  HeadingExample,
+  ButtonsExample,
+  AvatarExample,
+  IconExample,
+  CardExample,
+  TableExample,
+} from '../../../component-examples/SkeletonLoaderExamples';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/skeleton-loader"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/components/skeleton-loader/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Skeleton Loader"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Skeleton Loader"
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/skeleton-loader"
+  storybookUrl="/?path=/story/components-skeleton-loader--default"
+  data={props.data.allPasteComponent.edges}
+  packageStatus={props.data.allAirtable.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Guidelines
+
+### About Skeleton Loader
+
+Skeleton Loader displays a placeholder of a page's content before the data gets loaded. This helps reduce load-time frustration by making it feel like loading is happening immediately.
+
+It's an alternative to the traditionaly spinner loading methods. Rather than showing an abstract spinner, it creates anticipation of whats to come and reduces congitive load.
```

Comment:
```suggestion
The Skeleton Loader displays a placeholder representing the page's content before the page is ready to display. This component is used to improve [perceived performance](https://developer.mozilla.org/en-US/docs/Learn/Performance/Perceived_performance). This tends to improves user experience by reducing load time frustration and making the page feel more responsive.

This loader is an alternative to the [Spinner component](/components/spinner). Rather than showing an abstract spinner, it frames the content of what is to come which creates anticipation.
```

Reply:
Took a stab at clarifying and mentioning perceived performance.